src/js/bookmarks.js

```
@@ -0,0 +1,314 @@
+        <a href="#orgs" class="mt-2 inline-flex items-center gap-2 px-6 py-2.5 bg-primary text-white rounded-full text-sm font-bold hover:bg-orange-600 transition-colors shadow-md">
+          <span class="material-symbols-outlined text-sm">search</span> Browse Organizations
+        </a>
+      </div>`;
+    return;
+  }
+
+  grid.innerHTML = orgs
+    .map((org, i) => {
+      const comp = COMP_COLORS[org.competition] || COMP_COLORS.moderate;
+      const gh = org.github.split("/")[0];
+      const logoSrc = `https://github.com/${escapeHtml(gh)}.png?size=80`;
```

ESLint

[error] 'escapeHtml' is not defined.

(no-undef)